tests/features/workflows.feature

```diff
@@ -0,0 +1,11 @@
+Feature: workflows
+
+  Scenario: Workflows List
+    @vcr workflows-list
```

Comment:
If this is being rebased atop the Behat 3 upgrade and fixture name standardization, this should be `workflows_list` and its fixture file also renamed.